homeassistant/components/weather/ecobee.py

```diff
@@ -63,12 +62,18 @@ def name(self):
     @property
     def condition(self):
         """Return the current condition."""
-        return self.get_forecast(0, 'condition')
+        try:
+            return self.get_forecast(0, 'condition')
+        except ValueError:
+            return STATE_UNKNOWN
```

Comment:
Usually unknown attributes are not populated in the state attributes at all. So if we want to follow that, we should return `None` in that case.